src/main/java/org/gridsuite/modification/server/modifications/VscModification.java

```diff
@@ -31,14 +30,20 @@
  */
 
 public class VscModification extends AbstractModification {
+    public static final String NO_VALUE = "No value";
+    public static final String DROOP_FIELD = "Droop";
+    public static final String DROOP_AND_P0_FIELD = "Droop and P0";
+    public static final String P0_FIELD = "P0";
+    public static final String ACTIVE_POWER_CONTROL_EXTENSION_CREATE_ERROR_MESSAGE = "Both %s are required when angle droop active power control is activated to modify the equipment with id=%s";
```

Comment:
YES!
I have remove the last part of message in modification which contains the id of equipment, I think we do not need the id at this level because the report of the whole modification contains already the id.

Now we can share the message between creation and modification.

`    public static final String DROOP_ACTIVE_POWER_CONTROL_P0_DROOP_REQUIRED_ERROR_MSG = "Both Droop and P0 are required when angle droop active power control is activated";`

Reply:
DONE